src/css/style.css

```diff
@@ -1,3 +1,15 @@
 .is-vis-hidden{
     visibility: hidden;
+}
+
+@media (min-width: 1024px) {
+    .filters {
+        position: relative;
+        top: -40px;
+    }
+}
+
+
+.navbar {
+    background-color: transparent;
```

Comment:
why don't you add dedicated selector instead of playing with app wide and framework wide selector

Reply:
@alkeicam done